README.md

```diff
@@ -168,3 +168,40 @@ The following are the high level steps to update the notices page:
 1. Install [LicensePlist](https://github.com/mono0926/LicensePlist) `brew install licenseplist`
 2. Run `license-plist --markdown-path Acknowledgements.markdown`
 3. Open an issue on [Bedrock](https://github.com/mozilla/bedrock/) with a description and the requested page to update with a link to a document of the generated `Acknowledgements.markdown`
+
+## Releasing
+
+To release a version of Pocket following our every 2 week cycle you should follow the following depending on your release scenario. The following assumes you will be releasing to the Testflight Audience and then after some time promoting that version of the App to the store.
+
+If you need to increment the app version:
+1. If needed pr to increment the app version number in Git on the `develop` branch.
+2. Trigger the `nightly-internal-pipeline` manually to submit a new build to Testflight.
+
+If you will use the app version already in `develop`
+1. Find the nightly you will promote that is already Testflight Nightlies. Correlate it's build number with Bitrise and find the respective commit and write that down. You can search by build number in the Bitrise UI.
+   a. Alternatively trigger the `nightly-internal-pipeline` manually to submit a new build to Testflight from the latest on the `develop` branch.
+
+Once you have the build and associated commit you want to release to the Testflight audience perform the following in Git and Github:
+1. Tag the commit in git with the name `release/v0.0.0.0000` where 0.0.0 is the app version, and .0000 is the build number.
+2. Push the tag to github.
+3. Draft a new GitHub release by [clicking here](https://github.com/Pocket/pocket-ios/releases/new)
+4. For the name put `v0.0.0.0000` where 0.0.0 is the app version, and .0000 is the build number.
+5. In the Choose a tag dropdown select your tag you pushed to GitHub, and make sure the target is main. This will draft the release by comparing the changes from the Tag since the last release.
```

Comment:
```suggestion
5. In the 'Choose a tag' dropdown select your tag you pushed to GitHub, and make sure the target is main. This will draft the release by comparing the changes from the Tag since the last release.
```